### PR TITLE
Edits and comments

### PR DESCRIPTION
**Pull request by jchodera**
I took a quick pass through Significance and made some edits/comments.

### CODE REVIEW

**Comment by jchodera** [comment]
Working through Innovation now...

**Comment by jchodera** [comment]
I have to step away for a bit, so feel free to merge if you want to do more.

**Comment by davidlmobley** [comment]
Weekends are unpredictable and busy here; have been tied up all day so far with kid stuff and I have a couple other things ahead of this in the queue, then wrangling kids for dinner, bedtime, etc. So long story short -- may not even get to look over your changes today (though I hope to); certainly won't get to writing any new content til Monday.

**Comment by jchodera** [comment]
No problem! Just wanted to keep you updated so I didn't hold anything up
unnecessarily.

**Comment by davidlmobley** [comment]
Scrolling through the first few edits; generally looking good and quite helpful. It will probably be Monday before I go through everything in full detail though.

I'll make some comments on the TeX as I go in this PR for discussion.

File: latex/Research_Strategy.tex [hunk 1]
```diff
@@ -138,27 +138,29 @@
...
+For example, if the binding mode of a ligand is already known crystallographically, a researcher may use that binding mode in retrospective tests, whereas prospective or design work would require first selecting among candidate binding modes, introducing substantial uncertainty unaccounted for in the retrospective statistics~\cite{mobley_predicting_2007, boyce_predicting_2009, mobley_perspective_2012}.
 This also means that in retrospective tests, researchers almost invariably try far fewer methods than in prospective tests, resulting in much less new insight.
-Prospective application, while important, also does not provide the necessary impetus, partly because often, the predicted compounds are in fact never tested~\cite{christ_binding_2016} or the experimental data necessary to assess the quality of the predictions is absent -- for example, because binding affinities are not measured or no crystallography is available. 
+{\color{red}[JDC: Not sure I buy that---you can easily overfit in retrospective tests by trying many approaches to improve your score.]}
```
Comment (davidlmobley) [inline latex/Research_Strategy.tex:160]: Totally agree. I was making a different point, and probably there is a better way to say it, so I could use your help. The point I was making is this -- if you already know that a particular system is "well behaved", you know the relevant binding modes, etc., it's easy to just say, "oh, well, we don't need to worry about THOSE issues here" and so people often apply rather simple protocols in retrospective tests. Sure, they might apply a variety of simple variations of those protocols until they get the right answer, thereby overfitting, as you're pointing out. But, they tend to take a simple, straightforward approach.

What we tend to see in SAMPL, instead, is that people try a whole bunch of more complicated protocols because they are so uncertain about what the issues/challenges might even be. (People may try different binding modes of a ligand, or different protonation states, or different charge sets, etc.)

Anyway, that's more detail than we need to get into here. Would love thoughts on the best way to sell why prospective is so much better than retrospective, clearly. (Even Robert Abel is not really convinced of this.)

**Comment by jchodera** [comment]
Sounds good. I should have more time Sunday.

File: latex/Research_Strategy.tex [hunk 2]
```diff
@@ -167,17 +169,20 @@
...
-SAMPL is modeled as a \emph{challenge} rather than a \emph{competition}, with a key goal being to maximize what is learned rather than to declare winners and losers, with the idea that learning will provide the greatest long-term rewards in terms of progress in the field.
+One model for targeted blind challenges in physical modeling, the {\bf Statistical Assessment of Modeling of Proteins and Ligands (SAMPL)}, provides a highly successful framework that we here propose to extend and expand to accelerate progress in predictive physical modeling.
+Borne out of frustration with the lack of venues for comparing predictive accuracy on a level playing field for which current-generation models \emph{claimed} to be successful in retrospective work, SAMPL was initiated by Anthony Nicholls of OpenEye software in 2007/2008~\cite{nicholls_predicting_2008}, and has run approximately every two years since then~\cite{nicholls_samp1_2009, mobley_predictions_2009, geballe_sampl2_2010, geballe_sampl3_2012, mobley_blind_2014-1, muddana_sampl4_2014, bannan_blind_2016, yin_sampl5_preprint}.
```
Comment (davidlmobley) [inline latex/Research_Strategy.tex]: "Born"; "borne" is "carried by".

File: latex/Research_Strategy.tex [hunk 3]
```diff
@@ -167,17 +169,20 @@
...
 SAMPL has always involved a component focused on calculation of relatively straightforward physical properties such as hydration free energies, but also introduced host-guest systems as model binding systems for SAMPL3-5, supplementing protein-ligand challenges (on trypsin and HIV integrase) which appeared in SAMPL3-4. 
 SAMPL has already been a tremendous resource for the community, resulting in roughly 100 publications (some are coming out as of this writing) which are typically cited 5-50 times or more each [refs].
+{\color{red}[JDC: We need to rework this paragraph to convey the main idea of \emph{why} this specific kind of competition is able to rapidly drive the field forward. Focusing on specific problems, curating challenges and datasets to address them, learning from success and failure in prospective challenges, and then curating datasets that become standard benchmarks collectively represent \emph{exactly} what we need to drive rapid progress in the field, and we've got the publications and specific success stories to prove it. How can we communicate that?]}
```
Comment (davidlmobley) [inline latex/Research_Strategy.tex:180]: I'll have to think on this on Monday.

File: latex/Research_Strategy.tex [hunk 4]
```diff
@@ -167,17 +169,20 @@
...
-and the D3R Grand Challenges on protein-ligand binding, which are a major source of consternation for the community so far~\cite{ignjatovic_binding-affinity_2016, deng_large_2016, sunseri_d3r_2016, gathiaka_d3r_2016}.
+Until now, this has been impossible, because SAMPL has been entirely unfunded; its very existence has required ``donation'' of data and time from various sources, hindering the ability to collect dataset specifically tailored to maximize their ability to drive improvements to modeling. 
+The incarnation of SAMPL proposed here will deliberately bridge the gap between calculations of simple physical properties like hydration---which can be calculated fairly accurately with today's methods~\cite{mobley_blind_2014-1}---and the D3R Grand Challenges on protein-ligand binding, which are a major source of consternation for the community so far~\cite{ignjatovic_binding-affinity_2016, deng_large_2016, sunseri_d3r_2016, gathiaka_d3r_2016}.
+{\color{red}[JDC: Can we be a bit clearer about what we mean by ``simple physical properties like hydration'''---maybe ``simple physical properties that isolate forcefield inaccuracies from sampling challenges''?]}
```
Comment (davidlmobley) [inline latex/Research_Strategy.tex:186]: Yes, that would be fine.

File: latex/Research_Strategy.tex [hunk 5]
```diff
@@ -167,17 +169,20 @@
...
 %Add full list of references here.
 
 Here, we continue the legacy of SAMPL, but design a new series of SAMPL challenges specifically to maximize learning value to the community.
-Until now, this has been impossible, because SAMPL has been entirely unfunded, so its existence has required ``donation'' of data from various sources rather than data collected specifically to drive improvements to modeling. 
-Funding of SAMPL will also allow SAMPL to deliberately bridge the gap between calculations of simple physical properties like hydration -- which can be calculated fairly accurately with today's methods~\cite{mobley_blind_2014-1} -- 
-and the D3R Grand Challenges on protein-ligand binding, which are a major source of consternation for the community so far~\cite{ignjatovic_binding-affinity_2016, deng_large_2016, sunseri_d3r_2016, gathiaka_d3r_2016}.
+Until now, this has been impossible, because SAMPL has been entirely unfunded; its very existence has required ``donation'' of data and time from various sources, hindering the ability to collect dataset specifically tailored to maximize their ability to drive improvements to modeling.
```
Comment (davidlmobley) [inline latex/Research_Strategy.tex]: Probably slightly too wordy now -- maybe "specifically tailored to drive..." is enough, not "specifically tailored to maximize their ability to drive...".

Also should be "to collect datasets", not "to collect dataset"

File: latex/Research_Strategy.tex [hunk 6]
```diff
@@ -196,9 +201,10 @@
 %SAMPL drives science in our groups and in the field
    %a. Give examples from our groups from prior SAMPLs
    %b. Examples from field in prior SAMPLs
-Blind predictive challenges -- and SAMPL in particular -- already play a key role in fostering innovation in the field, especially in the form of method development, testing, and hardening, as well as force field development. 
-Thus, this work, by continuing and extending SAMPL, will advance innovative new science. 
-It is worth briefly highlighting several historical examples, though far more are available in the SAMPL literature. 
+Blind predictive challenges---and SAMPL in particular---already play a key role in fostering innovation in the field, especially in the form of method development, evaluation, and hardening, as well as force field development. 
+By expanding SAMPL, will advance innovative new science. 
+{\color{red}[JDC: How is it innovative if it has been going on for a while? Can we point to the number of new methodology publications it has motivated, for example?]}
```
Comment (davidlmobley) [inline latex/Research_Strategy.tex:207]: The point I was trying to make is that it drives innovation. In other words, I don't think the innovation of this proposal is that it's going to do something fundamentally new, and I think we have to be honest about that. Rather, the innovation we want to promote is the innovation that will result from SAMPL.

Certainly it motivates new methodology, though I'm not actually sure we can easily turn this into a number of new methodology publications it has motivated. I certainly couldn't compile a list of them -- it would require going around and asking past SAMPL participants to help do so, or reading the probably 1000+ papers citing past SAMPL publications and categorizing them. :)

File: latex/Research_Strategy.tex [hunk 7]
```diff
@@ -213,8 +219,8 @@
 %b.1 Figure showing progression in hydration free energy accuracy across a couple of SAMPLs or comparing two SAMPLs?
 %
  %Fennell SEA, etc.
-Shifts in protonation state and tautomer proved particularly important in the recent SAMPL5 $\log D$ challenge[refs], as they presumably will be for protein-ligand binding as well.
-Host-guest binding studies have also been particularly important[ref benchmark sets],
+Shifts in protonation state and tautomer proved particularly important in the recent SAMPL5 $\log D$ challenge [refs], which functions as a much more tractable venue for exploring the role these effects play in more complex protein-ligand binding.
```
Comment (davidlmobley) [inline latex/Research_Strategy.tex]: I'll have to hit this sentence up in revisions -- I don't like the "much more tractable venue" phrasing at all but I get what you're trying to do, so I need to rephrase. (It's too late in the day for me to do a good job coming up with new phrasings at this point, sadly.)

File: latex/Research_Strategy.tex [hunk 8]
```diff
@@ -227,27 +233,29 @@
 %c. Mention uniqueness of SAMPL -- there are other predictive challenges (D3R, pKa coop, CASP) but none focused on driving quantitatively predictive protein-ligand modeling
 This work is also innovative because of the uniqueness of SAMPL.
 While there are other predictive challenges in the area of biomolecular modeling, such as D3R [ref], the pKa cooperative [ref], and CASP [ref],
+{\color{red}[JDC: Is CAPRI still a thing? Are there other docking or ligand binding prediction challenges?]}
```
Comment (davidlmobley) [inline latex/Research_Strategy.tex:237]: Good question. I don't know. We should ask Twitter!

File: latex/Research_Strategy.tex [hunk 9]
```diff
@@ -227,27 +233,29 @@
...
 %insert references
 none are specifically focused on driving quantitatively predictive protein-ligand modeling.
-SAMPL is unique because -- at least with the deliberate design plan we propose here -- it will be specifically tailored in order to drive improvements in modeling for biomolecular design.
+The SAMPL expansion we propose here is unique because it will be \emph{specifically designed} to drive improvements in modeling accuracy for biomolecular design, rather than simply plan an evaluative role on targets of pharmaceutical interest.
```
Comment (davidlmobley) [inline latex/Research_Strategy.tex]: Grammar issues, I think. Maybe, "The SAMPL expansion we propose here is unique because it will be \emph{specifically designed} to drive improvements in modeling accuracy for biomolecular design, rather than simply serving [maybe you meant "playing an evaluative role"?] to evaluate accuracy on targets of pharmaceutical interest."

File: latex/Research_Strategy.tex [hunk 10]
```diff
@@ -257,14 +265,16 @@
 %References
 Additionally, we try to draw attention to and promote analysis of model uncertainty (as distinct from statistical uncertainty) in calculated values [refs], as understanding the confidence levels of predictions is particularly important for guiding molecular design.
 %Cite model uncertainty stuff.
+{\color{red}[JDC: Tracking down \emph{why} models are failing and what particular problems still need attention has also been an extremely powerful aspect of SAMPL. Much of this is left to the participants, of course, but we should mention the work the organizers do in spotting patterns and providing a venue for the participants to explore these issues.]}
```
Comment (davidlmobley) [inline latex/Research_Strategy.tex:269]: Good point, I totally agree.

File: latex/Research_Strategy.tex [hunk 11]
```diff
@@ -257,14 +265,16 @@
...
+Our approach to systematically advancing modeling for biomolecular design involves collecting carefully targeted experimental datasets for challenges focusing on physical property prediction, host-guest binding, and protein-ligand binding that isolate individual limitations in quantitative physical modeling to solicit and evaluate multiple solutions from the community.
+Aims 1--3 focus on tailoring and generating this experimental data, while Aim 4 focuses on fielding annual SAMPL challenges.
+Aims 1--3 bring together multiple laboratories and both theorists and experimentalists: graduate students from the Mobley and Chodera laboratories are paired with well-equipped experimental groups in industry to collect physical property data (Aim 1); leading experimentalists in supramolecular chemistry Gibb and Isaacs work with theorist Mobley to perform host-guest affinity measurements (Aim 2); new approaches to the automated development and investigation of model protein-ligand systems are developed in the Chodera lab (Aim 3).
```
Comment (davidlmobley) [inline latex/Research_Strategy.tex:277]: Do you think a student from your group would be involved in doing this too, or would it be primarily my group? I was assuming your folks would be too busy with the work on Aim 3, so the student in my group would be doing Aims 1 and 4 and yours would be doing Aim 3 exclusively. But it doesn't have to be that way.

**Comment by davidlmobley** [comment]
@jchodera - great. Great work on this -- gives me very concrete stuff to work on when I return to it on Monday or early in the week; seems to be shaping up nicely. Please do the same on the rest as you have time! 

Probably I'll spend my initial time on Monday trying to get the other sections drafted, then go back and work my way through the stuff you've added here while you're editing my new material. That would put us on track to get a rough draft of the content out to some close colleagues/friends by around Wednesday or so. Then when I polish more, finish my figures, etc., we could get it to Marti Head and some others along those lines for final help.

You're still headed for jury duty on the 27th, right?

**Comment by jchodera** [comment]
> Please do the same on the rest as you have time!

Working through that today.

> Probably I'll spend my initial time on Monday trying to get the other sections drafted, then go back and work my way through the stuff you've added here while you're editing my new material. 

Sounds good.

> You're still headed for jury duty on the 27th, right?

Starting Mon 26 Sep, yes, so I won't be available or online during 9-5 EST, but otherwise available.

File: latex/Research_Strategy.tex (hunk 11 above)
Comment (jchodera) [reply latex/Research_Strategy.tex:277]: I've written all my materials/biosketches/etc. with the assumption that both of our groups would contribute students for Aim 1. I have 9 students that could participate, all have some degree of experimental experience, and we have a track record of this now, so I felt this would be a strength, since your students do not necessarily have experimental experience.

Only Mehtap Isik (TPCB student working on model systems) will be fully dedicated to Aim 3. She would likely not participate in the industry internships unless as a result, unless we had someone else take over the Aim 3 project temporarily.

File: latex/Research_Strategy.tex (hunk 9 above)
Comment (jchodera) [reply latex/Research_Strategy.tex]: Yes, thanks!

File: latex/Research_Strategy.tex (hunk 8 above)
Comment (jchodera) [reply latex/Research_Strategy.tex:237]: CAPRI seems to have [had an event in 2016](http://www.ebi.ac.uk/msd-srv/capri/) alongside CASP.

File: latex/Research_Strategy.tex (hunk 7 above)
Comment (jchodera) [reply latex/Research_Strategy.tex]: We need to communicate why it is useful to do physical property measurements that isolate specific physical effects from the more complex pharmacological target cases. I'm happy with any phrasing that communicates this clearly.

File: latex/Research_Strategy.tex (hunk 6 above)
Comment (jchodera) [reply latex/Research_Strategy.tex:207]: > I don't think the innovation of this proposal is that it's going to do something fundamentally new, and I think we have to be honest about that.

I think it does help if we stress that we're going beyond what was already done, and see if we can communicate WHY it the proposed expansion of SAMPL is innovative and necessary, lest they think we should just carry on as we have without funding.

**Comment by jchodera** [comment]
OK, I've finished an edits/comments pass.

**Comment by davidlmobley** [comment]
Merging so I can begin my pass.

File: latex/Research_Strategy.tex (hunk 11 above)
Comment (davidlmobley) [reply latex/Research_Strategy.tex:277]: For the record, I actually have one grad student who did joint computational/experimental work as an undergrad (including a great deal of crystallography work), another who had experimental industry wet lab experience for two years before grad school (Caitlin, you've met her);  and another current student who is a 50/50 grad student shared with an experimental group. So we'd be able to do this too.  But in any case, this sounds good -- I just didn't want to assume you were in on Aim 1 if you didn't think you had room for it.

**Comment by jchodera** [comment]
Great! Sell that experience!

I think it will be better if it's clear that both of our labs are available
to engage west and east coast pharma, as long as there is a clear
leadership plan.
